fv3core/decorators.py

```diff
@@ -337,8 +337,15 @@ class FixedOriginStencil:
     and using the normalized origins."""
 
     def __init__(self, func, origin, domain, **kwargs):
-        self.normalized_origin = gtscript.gt_definitions.normalize_origin_mapping(
-            origin
+        self.normalized_origin = (
```

Comment:
Does this support passing an origin per field?

Reply:
yes of course

Reply:
👍